### ISSUE TITLE
getTableReorderColumn let headerActions dissapear inside a relationManager

### ISSUE THREAD

**Issue by annejan89** [filamentphp/filament#3265]
### Package

filament/filament

### Package Version

v2.15.2

### Laravel Version

v9.22.0

### Livewire Version

v2.10.6

### PHP Version

v8.1.3

### Problem description

Using method below to sort table rows by given column.

  ```
  protected function getTableReorderColumn(): ?string
    {
        return 'sort';
    }
```

When using this inside a relationManager the CreateAction button in the header disappears. 

```
->headerActions([
                Tables\Actions\CreateAction::make(),
])
```

### Expected behavior

Without using the getTableReorderColumn() method the create button shows correctly.

```
->headerActions([
                Tables\Actions\CreateAction::make(),
])
```

### Steps to reproduce

Tried to overwrite the canCreate method inside the Relationmanager, but without any luck.
Guess that getTableReorderColumn doesn't respect and merge with existing headerActions.

**Comment by zepfietje** [comment]
@annejan89, does the button disappear while reordering only, or permanently?

**Comment by annejan89** [comment]
> @annejan89, does the button disappear while reordering only, or permanently?

Permanently

When reordering is turned off / default:
<img width="1269" alt="Schermafbeelding 2022-07-27 om 18 47 50" src="https://user-images.githubusercontent.com/10883495/181304163-86f19552-cfe3-48c1-aaae-bf687d5cd740.png">

While reordering:
<img width="1263" alt="Schermafbeelding 2022-07-27 om 18 47 05" src="https://user-images.githubusercontent.com/10883495/181303856-9c2768e3-ccb1-4a35-a9c3-5d7035e61d81.png">

After deleting the getTableReorderColumn() method:
<img width="1273" alt="Schermafbeelding 2022-07-27 om 18 49 24" src="https://user-images.githubusercontent.com/10883495/181304386-1fb7e609-ed12-4aed-a154-16b7d26f9c34.png">

**Comment by zepfietje** [comment]
@annejan89 how exactly are you configuring the reordering functionality? 👇

> Also, could you share the exact code you use to configure the reordering on the relation manager? Usually, you'd just chain reorderable('sort') on the table inside the table() method on the relation manager class.
> 
> If you could set up a repository which reproduces the problem, we're able to fix the issue much faster.

**Comment by annejan89** [comment]
Tried both ways, both cause the problem as described:

`class ShiftsRelationManager extends RelationManager {
`
....

```
    protected function getTableReorderColumn(): ?string
    {
        return 'sort';
    }
```
Using reorderable() within the table method:

```
 public static function table(Table $table): Table
    {
        return $table
            ->columns([
                TextColumn::make('name')->label('Naam'),
                BadgeColumn::make('status_badge')->label('Capaciteit')

            ])
            ->filters([
                //
            ])
            ->headerActions([
                Tables\Actions\CreateAction::make(),
            ])
            ->actions([
                Tables\Actions\EditAction::make(),
                Tables\Actions\DeleteAction::make(),
            ])
            ->bulkActions([
                Tables\Actions\DeleteBulkAction::make(),
            ])
            ->reorderable('sort');
    }
```

**Comment by zepfietje** [comment]
Thanks for the more detailed snippets, @annejan89.

I can't reproduce the problem. Header actions show just fine for me when configuring a table as reorderable. Could you set up a clean Laravel project with minimal code required to show the problem? If you share the repository URL here, we can have a look.

**Comment by martinmildner** [comment]
The same thing happens to me. As soon as I add **protected function getTableReorderColumn()** to the Relation Manager, the headerAction disappears.

**Comment by zepfietje** [comment]
I still can't reproduce the issue. As I asked before, could someone set up a reproduction repository?

**Comment by danharrin** [comment]
Are you all on the latest version? I think I fixed this 1 days ago in v2.15.3, the issue report is v2.15.2

**Comment by martinmildner** [comment]
thxxx, @danharrin! it's fixed in v2.15.3
